client/components/Notification.js

```diff
@@ -3,37 +3,37 @@ import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 
 class Notification extends React.Component {

```

Comment:
This could be declared as a function instead of class that extends another class but its not using any of its function/methods, just saying for speed